### PR TITLE
:whale: Configuracoes para docker de desenvolvimento para o frontend #10

### PR DESCRIPTION
**Pull request by IuryNogueira**
Condigurações para o ambiente de desenvolvimento

### CODE REVIEW

File: frontend/dev.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+RUN yarn install
+
+RUN echo -e "yarn dev" > /start.sh
+RUN chmod +x /start.sh
+
+WORKDIR /frontend
+
+EXPOSE 3000
+
+ENTRYPOINT /start.sh
```
Comment (edimossilva) [inline frontend/dev.Dockerfile:13]: add empty line

File: frontend/docker-compose.dev.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+services:
+    frontend:
+        build:
+            context: ./
+            dockerfile: dev.Dockerfile
+        volumes:
+            - ./:/frontend
+        ports:
+            - 3000:3000
+        restart: unless-stopped
```
Comment (edimossilva) [inline frontend/docker-compose.dev.yml:12]: add empty line

File: frontend/docker-compose.dev.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+version: '3.7'
+
+services:
+    frontend:
```
Comment (edimossilva) [inline frontend/docker-compose.dev.yml:4]: fix spacing

File: frontend/docker-compose.dev.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+version: '3.7'
```
Comment (edimossilva) [inline frontend/docker-compose.dev.yml:1]: docker-compose up doesn't find  `frontend/docker-compose.dev.yml`, should I so something different?
image

**Review by edimossilva — CHANGES_REQUESTED** [review]
Please, also add a readme with instructions about how to use it.

I tried to rename `docker-compose.dev.yml` to `docker-compose.yml` and the service goes up but I couldn't access on localhost:3000

File: frontend/docker-compose.dev.yml (hunk 4 above)
Comment (IuryNogueira) [reply frontend/docker-compose.dev.yml:1]: Added in README the instructions to run with other file, this case using docker-compose.dev.yml.

**Review by andersonmalheiro — APPROVED** [review]

**Pull request merged**